src/pages/import-contacts/@modify-data/form-component.tsx

```diff
@@ -48,53 +51,36 @@ const FormComponent = ({
       phone: data?.phone ?? '',
     },
   })
-
-  async function handleDelete() {
-    if (data && data._id) {
-      setIsSubmitting(true)
-      try {
-        const res = await Delete({ id: data._id }).unwrap()
-
-        if (res.status === 'fail') throw new Error(res.message)
-      } catch (error) {
-        console.log(error)
-      } finally {
-        setIsSubmitting(false)
-      }
-    }
-  }
+  // console.log(setIsSubmitting)
 
   async function handleModifyProducts(formData: z.infer<typeof formSchema>) {
-    setIsSubmitting(true)
-
-    // const payload: {
-    //   name: string
-    //   type: 'WHOLE' | 'MIXTURE'
-    //   ingredients?: string[]
-    //   _id?: string
-    //   is_active?: boolean
-    // } = {
-    //   name: formData.name,
-    //   type: formData.type as 'WHOLE' | 'MIXTURE',
-    //   ingredients: formData.ingredients?.map((ingredient) => ingredient._id),
-    // }
-
-    // if (formData.type === 'WHOLE') delete payload.ingredients
+    // setIsSubmitting(true)
```

Comment:
Error- setIsSubmitting is not a function

*Line is Commented* - Uncomment and check the erro